src/pages/Main/Main.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import './Main.scss';
+import { useNavigate } from 'react-router-dom';
+
+const Main = () => {
+  const navigate = useNavigate();
+
+  const goToLogin = () => {
+    navigate('/');
+  };
+
+  return (
+    <div className="main">
+      <button className="back" type="button" onClick={goToLogin}>
+        {' '}
+        &lt; &nbsp;&nbsp;뒤로
+      </button>
+      <h2>회원가입</h2>
+      <div className="basic">
+        <div className="information">기본정보</div>
+        <div className="required">필수 사항</div>
+      </div>
+      <div className="input_main">
+        <input type="text" placeholder="이메일"></input>
+        <input type="text" placeholder="비밀번호"></input>
+        <input type="text" placeholder="비밀번호 확인"></input>
```

Comment:
여는태그와 닫는태그 사이에 content가 없는경우에는 self closing tag.활용해주세요~
```suggestion
        <input type="text" placeholder="이메일" />
        <input type="text" placeholder="비밀번호"  />
        <input type="text" placeholder="비밀번호 확인" />
```